VisualPinball.Unity/Documentation~/creators-guide/tutorials/realistic-backglass/3-import-into-unity.md

```diff
@@ -0,0 +1,46 @@
+---
+uid: tutorial_backglass_3
+title: Realistic backglass - Import Into Unity
+description: How setup your backglass in Unity
+---
+
+# Import Into Unity
+
+## Step 1: Import the FBX
+
+Open your scene in Unity and navigate to where your table stores its Materials, Meshes and textures.  Drag and drop your backglass mesh into "Meshes" and your images into "Textures"
+
+> [!note]
+> We recommend storing models at `Assets/<Table Name>/Models`, and texture at `Assets/<Table Name>/Textures`. In this example the Tropic Fun backglass is being created in another table for demonstrative purposes.
+
+![Navigate to Table Folders](BringMeshAndpngsIntoUnity.jpg)
+
+## Step 2: Add a Mesh Filter
+
+In the Hierarchy window add a new element by right clicking and choosing "create empty". In this case we named it "Backglass".  Right click in the "Inspector" window and search for "Mesh Filter" and add this to the inspector window.  With the Mesh Filter highlighted search for the Blender backglass mesh that you made.  Grab the "cube" mesh from this and drag-drop it into the "mesh" slot in the mesh filter.   If the cube cannot be moved, select the top level (in this case TropicFun BG), right click and select "unpack".
+
+![Add Mesh Filter](ChooseCubeFromtfAsMesh.jpg)
+
+## Step 2: Add a Mesh Renderer
+
+Next right click in the "Inspector" window, search for "Mesh Renderer" and add this to the inspector window.
+
+![Add Mesh Renderer](AddMeshFilterAndMeshRenderer.jpg)
+
+## Step 4: Add a Translite Material
+
+Navigate to the "Translite" material and drag and drop this into the "Materials Element" box of the Mesh Renderer.
```

Comment:
So here you're referring to the material that was created during the import, right? I think good practice is to rely on a previous import but describe it in a way it also works for a build from scratch. So you'd have to create this material first.

Reply:
I don't think that there is a translite material that is stock to vpe?  I had to grab it from Volley and bring it in.  Also there is a difference in folder structure between what was generated for me when I imported the vpx GT and what Volley has.  For example I don't think that I have a "models" folder?